### PR TITLE
support dynamically page height changing

### LINKED ISSUE #8: support dynamically page height changing

### PR DESCRIPTION
**Pull request by resessh**
closes #8

### CODE REVIEW

File: specs/sticky.spec.js [hunk 1]
```diff
@@ -42,19 +43,18 @@ describe('Sticky', () => {
       }
     `});
     await page.addScriptTag({ path: 'lib/index.js' });
-    await page.evaluate(selector => {
+    await page.evaluate((selector) => {
```
Comment (RyoNkmr) [inline specs/sticky.spec.js]: 🙏  please change eslint rules sorry

File: specs/sticky.spec.js [hunk 2]
```diff
@@ -42,19 +43,18 @@ describe('Sticky', () => {
...
       const { Sticky } = StuckJs;
       const element = document.querySelector(selector);
       const sticky = new Sticky(element);
     }, target);
-  }, 10000);
+  });
```
Comment (RyoNkmr) [inline specs/sticky.spec.js]: [ASK] why did you remove `Timeout` for `beforeEach` with async function

File: specs/sticky.spec.js [hunk 3]
```diff
@@ -64,19 +64,99 @@ describe('Sticky', () => {
 
   describe('DOM mutations', () => {
     it('adds stuck data attribute on created', async () => {
-      const stuck = await page.$eval(target, el => el.dataset.stuck)
+      const stuck = await page.$eval(target, el => el.dataset.stuck);
```
Comment (RyoNkmr) [inline specs/sticky.spec.js]: 👍

File: src/sticky.js [hunk 4]
```diff
@@ -54,6 +54,11 @@ export default class Sticky {
     return this.$$floor || 0;
   }
 
+  set floor(value: number): number {
+    this.$$floor = value;
+    return value;
```
Comment (RyoNkmr) [inline src/sticky.js]: 🙏 setter should not return any values (sorry it's my fault)

File: specs/sticky.spec.js (hunk 2 above)
Comment (resessh) [reply specs/sticky.spec.js]: it's my mistake
